lib/bankscrap/bbva/bank.rb

```diff
@@ -130,6 +132,19 @@ def login
           'eai_password'   => @password
         }
         post(BASE_ENDPOINT + LOGIN_ENDPOINT, fields: params)
+
+        # We also need to initialize a session
+        with_headers('Content-Type' => 'application/json') do
+          post(SESSIONS_ENDPOINT, fields: {
+            consumerID: CONSUMER_ID
+          }.to_json)
+        end
+
+        # We need to extract the "tsec" header from the last response.
+        # As the Bankscrap core library doesn't expose the headers of each response
+        # we have to use Mechanize's HTTP client "current_page" method.
+        tsec = @http.current_page.response['tsec']
```

Comment:
BTW, regarding `tsec`, AFAIK this header was not present in the API before. I think it is not mandatory, i.e. the API will still return a valid result if as long as the other headers are present. However as we want to replicate the mobile app's requests as close as possible, I decided to include it.